src/ctap/storage.rs

```diff
@@ -563,6 +604,15 @@ impl PersistentStore {
     pub fn force_pin_change(&mut self) -> Result<(), Ctap2StatusCode> {
         Ok(self.store.insert(key::FORCE_PIN_CHANGE, &[])?)
     }
+
+    /// The size used for shards of large blobs.
+    ///
+    /// This value is constant during the lifetime of the device.
+    #[cfg(test)]
+    fn shard_size(&self) -> usize {
+        self.store.max_value_length()
+    }
```

Comment:
Do we still need this?

Reply:
The second commit reduces the test complexity to be independent of shard sizes. Doing that, we can remove this function. Probably makes sense after shard tests are in the store now, but if you think I reduced testing too much, let me know.